test/common/http/http2/http2_frame.h

```diff
@@ -26,56 +26,56 @@ class Http2Frame {
   static const char Preamble[25];
 
   enum class Type : uint8_t {
-    DATA = 0,
-    HEADERS,
-    PRIORITY,
-    RST_STREAM,
-    SETTINGS,
-    PUSH_PROMISE,
-    PING,
-    GOAWAY,
-    WINDOW_UPDATE,
-    CONTINUATION
+    Data = 0,
+    Headers,
+    Priority,
+    RstStream,
+    Settings,
+    PushPromise,
+    Ping,
+    GoAway,
+    WindowUpdate,
+    Continuation
   };
 
   enum class SettingsFlags : uint8_t {
-    NONE = 0,
-    ACK = 1,
+    None = 0,
+    Ack = 1,
   };
 
   enum class HeadersFlags : uint8_t {
-    NONE = 0,
-    END_STREAM = 1,
-    END_HEADERS = 4,
+    None = 0,
+    EndStream = 1,
+    EndHeaders = 4,
   };
 
   enum class DataFlags : uint8_t {
-    NONE = 0,
-    END_STREAM = 1,
+    None = 0,
+    EndStream = 1,
   };
 
   // See https://tools.ietf.org/html/rfc7541#appendix-A for static header indexes
   enum class StaticHeaderIndex : uint8_t {
-    UNKNOWN,
-    METHOD_GET = 2,
-    METHOD_POST = 3,
-    PATH = 4,
-    STATUS_200 = 8,
-    STATUS_404 = 13,
-    SCHEME_HTTPS = 7,
-    HOST = 38,
+    Unknown,
+    MethodGet = 2,
+    MethodPost = 3,
+    Path = 4,
+    Status200 = 8,
+    Status404 = 13,
+    SchemeHttps = 7,
+    Host = 38,
   };
 
-  enum class ResponseStatus { UNKNOWN, _200, _404 };
+  enum class ResponseStatus { UNKNOWN, Ok, NotFound };
```

Comment:
missed this UNKNOWN

Reply:
How is this not caught by clang-tidy?

Reply:
Great eye, let me find an answer. My initial guess is that perhaps clang-tidy has some kind of leniency in guessing if something is an acronym, even though we don't treat acronyms with special casing.